assets/src/design-system/components/checkbox/index.js

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import propTypes from 'prop-types';
+import styled, { css } from 'styled-components';
+/**
+ * Internal dependencies
+ */
+import { Checkmark } from '../../icons';
+
+const CheckboxContainer = styled.div(
+  ({ disabled, theme }) => css`
+    ${disabled &&
+    css`
+      opacity: 0.5;
+    `}
+
+    position: relative;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 24px;
+    width: 24px;
+    margin: 3px;
```

Comment:
Sure! I added it because the focus border is thicker than the normal border. When the checkbox is focused, the checkbox moves:

However, by adding 1px of margin while the input is unfocused we can keep the element in the same spot on the page.

|without the margin|with 1px margin (while unfocused)|
|--|--|
|![checkbox-without-margin](https://user-images.githubusercontent.com/22185279/105093739-70c12b00-5a60-11eb-9409-2b98a600e893.gif)|![checkbox-with-margin](https://user-images.githubusercontent.com/22185279/105093967-c72e6980-5a60-11eb-9d54-86aeb299e7be.gif)|

If this is weird though we can always take it out! Thoughts?